howto/monitor/collect-metrics.md

```diff
@@ -0,0 +1,231 @@
+       constraints: "cpu-cores=4 mem=8G root-disk=100G"
+     '1':
+       series: focal
+       constraints: "cpu-cores=8 mem=16G root-disk=200G"
+     '2':
+       series: focal
+       constraints: "cpu-cores=4 mem=4G root-disk=100G"
+   ```
+   [/Details]
+
+   [Details="For the `anbox-cloud` bundle"]
+   ```yaml
+   applications:
+     grafana-proxy:
+       charm: 'cs:haproxy'
+       num_units: 1
+       expose: true
+       options:
+         default_mode: tcp
+         peering_mode: active-active
+         ssl_cert: SELFSIGNED
+         ssl_key: SELFSIGNED
+         services: |
+           - service_name: app-grafana
+             service_host: "0.0.0.0"
+             service_port: 443
+             service_options:
+               - mode http
+               - http-request set-path %[path,regsub(^/grafana/?,/)]
+               - balance leastconn
+             server_options: check verify none inter 2000 rise 2 fall 5 maxconn 4096
+             crts: [DEFAULT]
+           - service_name: http_service
+             service_host: "0.0.0.0"
+             service_port: 80
+             service_options:
+               - mode http
+               - http-request redirect scheme https
+       to:
+         - '3'
+
+     prometheus:
+       charm: 'cs:prometheus2'
+       num_units: 1
+       to:
+         - '3'
+
+     ams-monitor:
+       charm: 'cs:telegraf'
+       options:
+         tags: region=cloud-0
+         prometheus_output_port: "20003"
+
+     lxd-monitor:
+       charm: 'cs:telegraf'
+       options:
+         tags: region=cloud-0
+         prometheus_output_port: "20004"
+
+     grafana:
+       charm: 'cs:grafana'
+       num_units: 1
+       options:
+         root_url: '%(protocol)s://%(domain)s:%(http_port)s/grafana'
+       to:
+         - '3'
+
+   relations:
+     - ['ams:prometheus', 'prometheus:scrape']
+     - ['grafana:grafana-source', 'prometheus:grafana-source']
+     - ['grafana-proxy:reverseproxy', 'grafana:website']
+     - ['lxd-monitor:prometheus-client', 'prometheus:target']
+     - ['lxd-monitor:juju-info', 'lxd:juju-info']
+     - ['ams-monitor:prometheus-client', 'prometheus:target']
+     - ['ams-monitor:juju-info', 'ams:juju-info']
+     - ['ams:grafana', 'grafana:dashboards']
+     - ['anbox-stream-gateway:prometheus', 'prometheus:scrape']
+     - ['anbox-stream-gateway:grafana', 'grafana:dashboards']
+
+   machines:
+     '0':
+       series: focal
+       constraints: "cpu-cores=4 mem=4G root-disk=100G"
+     '1':
+       series: focal
+       constraints: "cpu-cores=4 mem=8G root-disk=100G"
+     '2':
+       series: focal
+       constraints: "cpu-cores=8 mem=16G root-disk=200G"
+     '3':
+       series: focal
+       constraints: "cpu-cores=4 mem=4G root-disk=100G"
+   ```
+   [/Details]
+
+   [note type="information" status="Note"]For a production-level solution, replace the automatically generated self-signed SSL credentials with proper ones.[/note]
+1. Deploy Anbox Cloud with the overlay file.
```

Comment:
Should we add a note about people who already have anbox cloud deployed but want to add the monitoring stack afterwards ?
It's the same command (juju will run a delta between the existing deployment and the bundle and only deploy new components), but it might not be very intuitive